Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -421,6 +421,14 @@ void AsyncScrollingCoordinator::setMouseMovedInContentArea(ScrollableArea& scrol
     stateNode->setMouseMovedInContentArea(state);
 }
 
+void AsyncScrollingCoordinator::setLayerHostingContextIdentifier(ScrollingNodeID scrollingNodeID, Markable<LayerHostingContextIdentifier> identifier)
+{
+    auto stateNode = dynamicDowncast<ScrollingStateFrameHostingNode>(m_scrollingStateTree->stateNodeForID(scrollingNodeID));
+    if (!stateNode)
```

Comment:
This should assert, since it's an error to call this on something that isn't a frame hosting node.

Reply:
Fixed.